security/pkg/stsservice/tokenmanager/sts_test.go

```diff
@@ -161,16 +198,22 @@ func sendHTTPRequestWithRetry(client *http.Client, req *http.Request) (resp *htt
 	return resp, err
 }
 
+type testSetUp struct {
+	enableCache        bool
+	enableDynamicToken bool
```

Comment:
What does enableDynamicToken mean?

Reply:
So the fake token has a static prefix and timestamp, if two STS requests get different token, then the token from second response is not served from cache.